erts/doc/src/persistent_term.xml

```diff
@@ -255,6 +255,22 @@ will be slower as the number of persistent terms increases.</pre>
       </desc>
     </func>
 
+    <func>
+      <name name="get" arity="2" since="OTP 21.3"/>
+      <fsummary>Get the value for a persistent term.</fsummary>
+      <desc>
+        <p>Retrieve the value for the persistent term associated with
+        the key <c><anno>Key</anno></c>. The lookup will be made in
+	constant time and the value will not be copied to the heap
+	of the calling process.</p>
+	<p>This function returns <c><anno>Key</anno></c> if no
```

Comment:
returns **Default** is key does not exists.

Reply:
Thanks! Fixed.